contracts/cw20-stake/src/contract.rs

```diff
@@ -436,6 +436,39 @@ pub fn query_hooks(deps: Deps) -> StdResult<GetHooksResponse> {
     })
 }
 
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn migrate(deps: DepsMut, _env: Env, msg: MigrateMsg) -> Result<Response, ContractError> {
```

Comment:
This is great! A test would be nice? Though we will do some manual testing.

Reply:
Agree about the test. It's a bit of an operation to test properly in multi-test because we'd need to check in a beta version of the staking contract. Might be worthwhile though to do that and put it in `legacy/` then write some tests going through the whole migration process in cw-core.